autoscaler/main.go

```diff
@@ -92,7 +93,7 @@ func main() {
 			"Enabling this will ensure there is only one active controller manager.")
 	flag.StringVar(&imagePullSecretsString, "image-pull-secrets", "",
 		"The image pull secrets to use for the collectors created by autoscaler")
-	flag.StringVar(&nameutils.ImagePrefix, "image-prefix", "", "The image prefix to use for the collectors created by autoscaler")
+	flag.StringVar(&nameutils.ImagePrefix, "image-prefix", "", "The image prefix to use for the collectors created by autoscaler (DEPRECATED: Use ODIGOS_COLLECTOR_IMAGE env variable)")
```

Comment:
let's also cleanup where it is being populated:

https://github.com/odigos-io/odigos/blob/dfff83dc69f80f88d9d623bece56c8d05fc06f60/cli/cmd/resources/autoscaler.go#L355

Reply:
and also helm here:
https://github.com/odigos-io/odigos/blob/dfff83dc69f80f88d9d623bece56c8d05fc06f60/helm/odigos/templates/autoscaler/deployment.yaml#L26